lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -507,13 +507,12 @@ defmodule Kernel.SpecialForms do
   defmacro alias(module, opts), do: error!([module, opts])
 
   @doc """
-  Requires a given module to be compiled and loaded.
+  Tells Elixir you want to use the macros defined in a module.
```

Comment:
I see what you mean, but the short line of documentation at the top is meant to just remind you what a function does or give you a slight idea so that you can go to the full docs if you need to use it. We do the same for `alias` and `import` btw, where we mention aliasing and importing respectively.